python/learn.py

```diff
@@ -43,12 +43,13 @@ def get_progress():
     Options:
       --help                     Show this message.
       --curriculum=<file>        Curriculum json file for environment [default: None].
-      --slow                     Whether to run the game at training speed [default: False].
       --keep-checkpoints=<n>     How many model checkpoints to keep [default: 5].
       --lesson=<n>               Start learning from this lesson [default: 0].
       --load                     Whether to load the model or randomly initialize [default: False].
-      --run-path=<path>          The sub-directory name for model and summary statistics [default: ppo]. 
+      --run-id=<path>            The sub-directory name for model and summary statistics [default: ppo]. 
```

Comment:
If this is branching from dev-0.3, I think this is a typo and should be --run-path. model_path below uses --run-path anyways.